script/InviteGenerator.sol

```diff
@@ -26,12 +26,12 @@ contract InviteGenerator is Script {
   /// @notice Error for empty invite signature version
   error InvalidSignatureVersion();
 
-  constructor(string memory inviteSigningDomain, string memory inviteSignatureVersion, string memory structName) {
+  constructor(string memory inviteSigningDomain, string memory inviteSignatureVersion) {
```

Comment:
small nit is that i don't understand why you keep insisting to have the strings on chain 😵